Makefile

```diff
@@ -15,7 +15,7 @@ verify:
 clean: delete_mgmt_cluster host_cleanup
 
 delete_mgmt_cluster:
-	minikube delete || true
+	sudo minikube delete || true
```

Comment:
why not the `sudo su` approach on this one?

Reply:
there is no need for it here, this can be run as sudo.

Reply:
why does it need sudo at all, since we're still starting it as a regular user?